CN Status Heatmap: Temporarily add a chunk that puts the n = 1 in "Other" (#709)
AlexsLemonade/OpenPBTA-analysis: analyses/cnv-chrom-plot/cn_status_heatmap.Rmd

# Absolute fraction needed for a bin to be called a particular status
frac_threshold <- 0.75

> # Any groups this size or smaller will be added into the `Other` group for the resulting heatmap
> min_group_size <- 1

Reviewer: ```suggestion
# Any groups smaller than this will be added into the `Other` group for the resulting heatmap
min_group_size <- 2
```

  dplyr::group_by(short_histology) %>% 
  dplyr::mutate(group_n = dplyr::n()) %>% 
  # Ungroup the data
  dplyr::ungroup() %>% 
  # Temporarily we will put the n = 1 `short_histology` samples in the `Other` group. 
  #TODO: Remove lines 387 - 400 when this is split into two panels
  dplyr::mutate(
    short_histology = dplyr::case_when(

>     group_n == 1 ~ "Other", 
>     TRUE ~ as.character(short_histology))) %>% 

Reviewer: ```suggestion
      group_n == 1 ~ "Other", 
      TRUE ~ as.character(short_histology))) %>% 
```

>   dplyr::mutate(
>     sample_color = dplyr::case_when(
>     group_n == 1 ~ "#cc0052",
>     TRUE ~ as.character(sample_color))) %>% 

Reviewer: Use the same method to reset the colors as before, for flexibility and robustness
```suggestion
  dplyr::mutate(sample_color = dplyr::recode(
    short_histology,
    !!!histology_col_palette
  )) %>%
```

  dplyr::group_by(short_histology) %>% 
  dplyr::mutate(group_n = dplyr::n()) %>% 
  # Ungroup the data
  dplyr::ungroup() %>% 
  # Temporarily we will put the n = 1 `short_histology` samples in the `Other` group. 
  #TODO: Remove lines 387 - 400 when this is split into two panels
  dplyr::mutate(
    short_histology = dplyr::case_when(

>       group_n <= min_group_size ~ "Other", 

Reviewer: ```suggestion
      group_n < min_group_size ~ "Other", 
```
